src/Flexbox.jsx

```diff
@@ -1,39 +1,38 @@
-import React, { PropTypes } from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 import styled from 'styled-components';
 
-const { string, oneOf, node, number, bool, object } = PropTypes;
```

Comment:
That's fine, then we may want to use destructuring when importing the module and not as a separate statement.